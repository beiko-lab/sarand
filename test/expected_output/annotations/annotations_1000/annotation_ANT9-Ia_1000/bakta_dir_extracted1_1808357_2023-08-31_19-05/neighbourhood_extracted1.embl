ID   contig_1; ; linear; DNA; ; UNC; 2783 BP.
XX
AC   contig_1;
XX
DE   contig_1, whole genome shotgun sequence
XX
OS   .
OC   .
XX
CC   Annotated with Bakta
CC   Software: v1.8.1
CC   Database: v5.0, light
CC   DOI: 10.1099/mgen.0.000685
CC   URL: github.com/oschwengers/bakta
CC   
CC   ##Genome Annotation Summary:##
CC   Annotation Date                :: 08/31/2023, 19:06:00
CC   CDSs                           ::     4
CC   tRNAs                          ::     0
CC   tmRNAs                         ::     0
CC   rRNAs                          ::     0
CC   ncRNAs                         ::     0
CC   regulatory ncRNAs              ::     0
CC   CRISPR Arrays                  ::     0
CC   oriCs/oriVs                    ::     0
CC   oriTs                          ::     0
CC   gaps                           ::     0
CC   pseudogenes                    ::     0
XX
FH   Key             Location/Qualifiers
FH
FT   source          1..2783
FT                   /mol_type="genomic DNA"
FT   gene            11..466
FT                   /locus_tag="KIGIOP_00005"
FT   CDS             11..466
FT                   /product="hypothetical protein"
FT                   /locus_tag="KIGIOP_00005"
FT                   /protein_id="gnl|Bakta|KIGIOP_00005"
FT                   /translation="MPQHIVQKFLGHESPEMTSRYAHIFDETLKNEFTKFQEKLVTNNG
FT                   DVLDLDEDNEVDDVELQWFKKNINAQVLPNGYCRLPVVAGGCPHANACLDCTHFCTSKQ
FT                   FLPQHEEQLERTEELLAIAKDKQWQRQVETNSRVKERLEQIIGSLTG"
FT                   /codon_start=1
FT                   /transl_table=11
FT                   /inference="ab initio prediction:Prodigal:2.6"
FT   gene            473..850
FT                   /locus_tag="KIGIOP_00010"
FT   CDS             473..850
FT                   /product="hypothetical protein"
FT                   /locus_tag="KIGIOP_00010"
FT                   /protein_id="gnl|Bakta|KIGIOP_00010"
FT                   /translation="MDKQVRNTTEIVRLAKQKSKKTREKVDKAISKFSIEGKVINFNSI
FT                   AKEANVSKSWLYKEHDIRQRIESLRERQITANVVSKPKKSSRSEEILIKTLKRRVMELE
FT                   KENKKLQNQIQKLYGDLYNKE"
FT                   /codon_start=1
FT                   /transl_table=11
FT                   /inference="ab initio prediction:Prodigal:2.6"
FT   gene            1001..1783
FT                   /locus_tag="KIGIOP_00015"
FT                   /gene="ant(9)-Ia"
FT   CDS             1001..1783
FT                   /db_xref="NCBIProtein:WP_000067268.1"
FT                   /db_xref="SO:0001217"
FT                   /db_xref="UniRef:UniRef50_P0A0D0"
FT                   /product="aminoglycoside nucleotidyltransferase ANT(9)-Ia"
FT                   /locus_tag="KIGIOP_00015"
FT                   /protein_id="gnl|Bakta|KIGIOP_00015"
FT                   /translation="MSNLINGKIPNQAIQTLKIVKDLFGSSIVGVYLFGSAVNGGLRIN
FT                   SDVDVLVVVNHSLPQLTRKKLTERLMTISGKIGNTDSVRPLEVTVINRSEVVPWQYPPK
FT                   REFIYGEWLRGEFENGQIQEPSYDPDLAIVLAQARKNSISLFGPDSSSILVSVPLTDIR
FT                   RAIKDSLPELIEGIKGDERNVILTLARMWQTVTTGEITSKDVAAEWAIPLLPKEHVTLL
FT                   DIARKGYRGECDDKWEGLYSKVKALVKYMKNSIETSLN"
FT                   /codon_start=1
FT                   /transl_table=11
FT                   /inference="ab initio prediction:Prodigal:2.6"
FT                   /inference="similar to AA sequence:UniRef:UniRef50_P0A0D0"
FT                   /gene="ant(9)-Ia"
FT   gene            complement(1909..2640)
FT                   /locus_tag="KIGIOP_00020"
FT                   /gene="erm(A)"
FT   CDS             complement(1909..2640)
FT                   /db_xref="NCBIProtein:WP_001072201.1"
FT                   /db_xref="SO:0001217"
FT                   /db_xref="UniRef:UniRef50_P13956"
FT                   /product="23S rRNA (adenine(2058)-N(6))-methyltransferase
FT                   Erm(A)"
FT                   /locus_tag="KIGIOP_00020"
FT                   /protein_id="gnl|Bakta|KIGIOP_00020"
FT                   /translation="MNQKNPKDTQNFITSKKHVKEILNHTNISKQDNVIEIGSGKGHFT
FT                   KELVKMSRSVTAIEIDGGLCQVTKEAVNPSENIKVIQTDILKFSFPKHINYKIYGNIPY
FT                   NISTDIVKRITFESQAKYSYLIVEKGFAKRLQNLQRALGLLLMVEMDIKMLKKVPPLYF
FT                   HPKPSVDSVLIVLERHQPLISKKDYKKYRSFVYKWVNREYRVLFTKNQFRQALKHANVT
FT                   NINKLSKEQFLSIFNSYKLFH"
FT                   /codon_start=1
FT                   /transl_table=11
FT                   /inference="ab initio prediction:Prodigal:2.6"
FT                   /inference="similar to AA sequence:UniRef:UniRef50_P13956"
FT                   /gene="erm(A)"
XX
SQ   Sequence 2783 BP; 975 A; 417 C; 518 G; 873 T; 0 other;
     taacaacggg atgccccagc atattgtgca gaaatttttg gggcatgaaa gcccagaaat        60
     gacaagcaga tacgctcata tctttgatga aactctaaaa aatgaattta ctaaatttca       120
     ggaaaaactg gttaccaata atggagatgt gcttgatcta gatgaagata atgaagtcga       180
     tgatgtagag cttcaatggt tcaagaaaaa tataaatgca caagtgcttc caaatggtta       240
     ttgtagattg ccagtagtag caggtggttg tccacatgcg aatgcatgct tagattgcac       300
     tcacttctgt accagtaagc aattcttacc acagcacgaa gaacagttag agcgtacaga       360
     agagttatta gccatagcta aggataaaca atggcaaaga caagtagaga ctaatagccg       420
     tgttaaagag cgtttagaac aaatcattgg aagtttgacg gggtaattat caatggataa       480
     acaagttaga aatacaacag aaattgtacg tttggcgaag cagaaatcaa aaaagacaag       540
     ggaaaaagta gacaaagcga tttctaaatt ttcgattgaa ggtaaagtta ttaattttaa       600
     ttcaatagca aaggaagcta atgtttctaa atcatggctt tataaggaac acgatattag       660
     gcaaagaatc gaatcccttc gtgagcgtca aataacagca aatgtagtct caaaacccaa       720
     gaaaagttct cgttcggagg aaatccttat taaaacctta aaaagaagag taatggaatt       780
     agaaaaagaa aataaaaaat tacagaacca aattcaaaaa ttatatggag atctgtataa       840
     taaagaataa ttattaatct gtagacaaat tgtgaaagga tgtacttaaa cgctaacggt       900
     cagctttatt gaacagtaat ttaagtatat gtccaatcta gggtaagtaa attgagtatc       960
     aatataaact ttatatgaac ataatcaacg aggtgaaatc atgagcaatt tgattaacgg      1020
     aaaaatacca aatcaagcga ttcaaacatt aaaaatcgta aaagatttat ttggaagttc      1080
     aatagttgga gtatatctat ttggttcagc agtaaatggt ggtttacgca ttaacagcga      1140
     tgtagatgtt ctagtcgtcg tgaatcatag tttacctcaa ttaactcgaa aaaaactaac      1200
     agaaagacta atgactatat caggaaagat tggaaatacg gattctgtta gaccacttga      1260
     agttacggtt ataaatagga gtgaagttgt cccttggcaa tatcctccaa aaagagaatt      1320
     tatatacggt gagtggctca ggggtgaatt tgagaatgga caaattcagg aaccaagcta      1380
     tgatcctgat ttggctattg ttttagcaca agcaagaaag aatagtattt ctctatttgg      1440
     tcctgattct tcaagtatac ttgtctccgt acctttgaca gatattcgaa gagcaattaa      1500
     ggattctttg ccagaactaa ttgaggggat aaaaggtgat gagcgtaatg taattttaac      1560
     cctagctcga atgtggcaaa cagtgactac tggtgaaatt acctcgaaag atgtcgctgc      1620
     agaatgggct atacctcttt tacctaaaga gcatgtaact ttactggata tagctagaaa      1680
     aggctatcgg ggagagtgtg atgataagtg ggaaggacta tattcaaagg tgaaagcact      1740
     cgttaagtat atgaaaaatt ctatagaaac ttctctcaat taggctaatt ttattgcaat      1800
     aacaggtgct tacttttaaa actactgatt tattgataaa tattgaacaa tttttgggaa      1860
     gaataaagcg tcctcttgtg aaattagaga acgctttatt actttaattt agtgaaacaa      1920
     tttgtaacta ttgaaaatag aaagaaattg ttccttcgat agtttattaa tattagtgac      1980
     atttgcatgc ttcaaagcct gtcggaattg gtttttagtg aaaagaacac gatattcacg      2040
     gtttacccac ttataaacaa aagatcgata ctttttgtag tccttctttg aaatcaatgg      2100
     ttgatgtcgt tcaagaacaa tcaatacaga gtctacactt ggcttaggat gaaaatatag      2160
     tggtggtact tttttgagca tttttatatc catctccacc attaatagta aacccaaagc      2220
     tcgttgcaga ttttgcaatc ttttcgcaaa tcccttctca acgataagat agctatattt      2280
     agcctgactt tcaaaggtaa ttcttttgac aatatccgta ctgatgttat aaggaatatt      2340
     accatatatc ttatagttta tatgttttgg gaaggaaaat tttagaatat ccgtttgaat      2400
     cacttttata ttctcagagg ggtttaccgc ttctttagtc acttgacata agcctccatc      2460
     aatttctata gcagtaactg atcgactcat tttgactagc tctttggtaa aatgtccttt      2520
     tcctgatccg atttctatta cgttgtcttg tttactgata ttcgtgtgat tcaatatttc      2580
     ttttacatgc tttttagaag taataaaatt ttgcgtgtct ttagggtttt tctggttcat      2640
     tataaccttc tccttactgg ttcattataa ccttctcctt actggttata atgaactgac      2700
     attaagcgtt cattataacc aattaatttt gatttggttg ataacgaact ttattaataa      2760
     caaatataga aaaagtaccc ata                                              2783
//
