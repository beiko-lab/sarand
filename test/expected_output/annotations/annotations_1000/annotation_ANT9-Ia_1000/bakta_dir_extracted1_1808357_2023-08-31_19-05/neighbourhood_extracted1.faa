>KIGIOP_00005 hypothetical protein
MPQHIVQKFLGHESPEMTSRYAHIFDETLKNEFTKFQEKLVTNNGDVLDLDEDNEVDDVELQWFKKNINAQVLPNGYCRLPVVAGGCPHANACLDCTHFCTSKQFLPQHEEQLERTEELLAIAKDKQWQRQVETNSRVKERLEQIIGSLTG
>KIGIOP_00010 hypothetical protein
MDKQVRNTTEIVRLAKQKSKKTREKVDKAISKFSIEGKVINFNSIAKEANVSKSWLYKEHDIRQRIESLRERQITANVVSKPKKSSRSEEILIKTLKRRVMELEKENKKLQNQIQKLYGDLYNKE
>KIGIOP_00015 aminoglycoside nucleotidyltransferase ANT(9)-Ia
MSNLINGKIPNQAIQTLKIVKDLFGSSIVGVYLFGSAVNGGLRINSDVDVLVVVNHSLPQLTRKKLTERLMTISGKIGNTDSVRPLEVTVINRSEVVPWQYPPKREFIYGEWLRGEFENGQIQEPSYDPDLAIVLAQARKNSISLFGPDSSSILVSVPLTDIRRAIKDSLPELIEGIKGDERNVILTLARMWQTVTTGEITSKDVAAEWAIPLLPKEHVTLLDIARKGYRGECDDKWEGLYSKVKALVKYMKNSIETSLN
>KIGIOP_00020 23S rRNA (adenine(2058)-N(6))-methyltransferase Erm(A)
MNQKNPKDTQNFITSKKHVKEILNHTNISKQDNVIEIGSGKGHFTKELVKMSRSVTAIEIDGGLCQVTKEAVNPSENIKVIQTDILKFSFPKHINYKIYGNIPYNISTDIVKRITFESQAKYSYLIVEKGFAKRLQNLQRALGLLLMVEMDIKMLKKVPPLYFHPKPSVDSVLIVLERHQPLISKKDYKKYRSFVYKWVNREYRVLFTKNQFRQALKHANVTNINKLSKEQFLSIFNSYKLFH
