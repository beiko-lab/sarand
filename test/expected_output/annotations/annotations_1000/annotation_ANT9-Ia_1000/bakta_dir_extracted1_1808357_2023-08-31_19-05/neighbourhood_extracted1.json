{
    "genome": {
        "genus": null,
        "species": null,
        "strain": null,
        "complete": false,
        "gram": "?",
        "translation_table": 11
    },
    "stats": {
        "no_sequences": 1,
        "size": 2783,
        "gc": 0.3359683794466403,
        "n_ratio": 0.0,
        "n50": 2783,
        "coding_ratio": 0.8440531800215595
    },
    "features": [
        {
            "type": "cds",
            "contig": "contig_1",
            "start": 11,
            "stop": 466,
            "strand": "+",
            "gene": null,
            "product": "hypothetical protein",
            "start_type": "ATG",
            "rbs_motif": null,
            "db_xrefs": [],
            "frame": 2,
            "aa": "MPQHIVQKFLGHESPEMTSRYAHIFDETLKNEFTKFQEKLVTNNGDVLDLDEDNEVDDVELQWFKKNINAQVLPNGYCRLPVVAGGCPHANACLDCTHFCTSKQFLPQHEEQLERTEELLAIAKDKQWQRQVETNSRVKERLEQIIGSLTG",
            "aa_hexdigest": "44d0fbaace285a4bfb4d3820eb132f1f",
            "nt": "ATGCCCCAGCATATTGTGCAGAAATTTTTGGGGCATGAAAGCCCAGAAATGACAAGCAGATACGCTCATATCTTTGATGAAACTCTAAAAAATGAATTTACTAAATTTCAGGAAAAACTGGTTACCAATAATGGAGATGTGCTTGATCTAGATGAAGATAATGAAGTCGATGATGTAGAGCTTCAATGGTTCAAGAAAAATATAAATGCACAAGTGCTTCCAAATGGTTATTGTAGATTGCCAGTAGTAGCAGGTGGTTGTCCACATGCGAATGCATGCTTAGATTGCACTCACTTCTGTACCAGTAAGCAATTCTTACCACAGCACGAAGAACAGTTAGAGCGTACAGAAGAGTTATTAGCCATAGCTAAGGATAAACAATGGCAAAGACAAGTAGAGACTAATAGCCGTGTTAAAGAGCGTTTAGAACAAATCATTGGAAGTTTGACGGGGTAA",
            "hypothetical": true,
            "genes": [],
            "seq_stats": {
                "molecular_weight": 17425.398799999995,
                "isoelectric_point": 5.097514152526855
            },
            "id": "KIGIOPMKKF_1",
            "locus": "KIGIOP_00005"
        },
        {
            "type": "cds",
            "contig": "contig_1",
            "start": 473,
            "stop": 850,
            "strand": "+",
            "gene": null,
            "product": "hypothetical protein",
            "start_type": "ATG",
            "rbs_motif": null,
            "db_xrefs": [],
            "frame": 2,
            "aa": "MDKQVRNTTEIVRLAKQKSKKTREKVDKAISKFSIEGKVINFNSIAKEANVSKSWLYKEHDIRQRIESLRERQITANVVSKPKKSSRSEEILIKTLKRRVMELEKENKKLQNQIQKLYGDLYNKE",
            "aa_hexdigest": "5505404e69595126558614f905b553d0",
            "nt": "ATGGATAAACAAGTTAGAAATACAACAGAAATTGTACGTTTGGCGAAGCAGAAATCAAAAAAGACAAGGGAAAAAGTAGACAAAGCGATTTCTAAATTTTCGATTGAAGGTAAAGTTATTAATTTTAATTCAATAGCAAAGGAAGCTAATGTTTCTAAATCATGGCTTTATAAGGAACACGATATTAGGCAAAGAATCGAATCCCTTCGTGAGCGTCAAATAACAGCAAATGTAGTCTCAAAACCCAAGAAAAGTTCTCGTTCGGAGGAAATCCTTATTAAAACCTTAAAAAGAAGAGTAATGGAATTAGAAAAAGAAAATAAAAAATTACAGAACCAAATTCAAAAATTATATGGAGATCTGTATAATAAAGAATAA",
            "hypothetical": true,
            "genes": [],
            "seq_stats": {
                "molecular_weight": 14804.005199999989,
                "isoelectric_point": 10.096151924133299
            },
            "id": "KIGIOPMKKF_2",
            "locus": "KIGIOP_00010"
        },
        {
            "type": "cds",
            "contig": "contig_1",
            "start": 1001,
            "stop": 1783,
            "strand": "+",
            "gene": "ant(9)-Ia",
            "product": "aminoglycoside nucleotidyltransferase ANT(9)-Ia",
            "start_type": "ATG",
            "rbs_motif": "GGAG/GAGG",
            "db_xrefs": [
                "NCBIProtein:WP_000067268.1",
                "SO:0001217",
                "UniRef:UniRef50_P0A0D0"
            ],
            "frame": 2,
            "aa": "MSNLINGKIPNQAIQTLKIVKDLFGSSIVGVYLFGSAVNGGLRINSDVDVLVVVNHSLPQLTRKKLTERLMTISGKIGNTDSVRPLEVTVINRSEVVPWQYPPKREFIYGEWLRGEFENGQIQEPSYDPDLAIVLAQARKNSISLFGPDSSSILVSVPLTDIRRAIKDSLPELIEGIKGDERNVILTLARMWQTVTTGEITSKDVAAEWAIPLLPKEHVTLLDIARKGYRGECDDKWEGLYSKVKALVKYMKNSIETSLN",
            "aa_hexdigest": "3a005c5d9c3ea91391713fea7aede43a",
            "nt": "ATGAGCAATTTGATTAACGGAAAAATACCAAATCAAGCGATTCAAACATTAAAAATCGTAAAAGATTTATTTGGAAGTTCAATAGTTGGAGTATATCTATTTGGTTCAGCAGTAAATGGTGGTTTACGCATTAACAGCGATGTAGATGTTCTAGTCGTCGTGAATCATAGTTTACCTCAATTAACTCGAAAAAAACTAACAGAAAGACTAATGACTATATCAGGAAAGATTGGAAATACGGATTCTGTTAGACCACTTGAAGTTACGGTTATAAATAGGAGTGAAGTTGTCCCTTGGCAATATCCTCCAAAAAGAGAATTTATATACGGTGAGTGGCTCAGGGGTGAATTTGAGAATGGACAAATTCAGGAACCAAGCTATGATCCTGATTTGGCTATTGTTTTAGCACAAGCAAGAAAGAATAGTATTTCTCTATTTGGTCCTGATTCTTCAAGTATACTTGTCTCCGTACCTTTGACAGATATTCGAAGAGCAATTAAGGATTCTTTGCCAGAACTAATTGAGGGGATAAAAGGTGATGAGCGTAATGTAATTTTAACCCTAGCTCGAATGTGGCAAACAGTGACTACTGGTGAAATTACCTCGAAAGATGTCGCTGCAGAATGGGCTATACCTCTTTTACCTAAAGAGCATGTAACTTTACTGGATATAGCTAGAAAAGGCTATCGGGGAGAGTGTGATGATAAGTGGGAAGGACTATATTCAAAGGTGAAAGCACTCGTTAAGTATATGAAAAATTCTATAGAAACTTCTCTCAATTAG",
            "pscc": {
                "uniref50_id": "UniRef50_P0A0D0",
                "db_xrefs": [
                    "SO:0001217",
                    "UniRef:UniRef50_P0A0D0"
                ],
                "product": "Spectinomycin 9-adenylyltransferase"
            },
            "expert": [
                {
                    "type": "amrfinder",
                    "rank": 95,
                    "gene": "ant(9)-Ia",
                    "product": "aminoglycoside nucleotidyltransferase ANT(9)-Ia",
                    "method": "EXACTP",
                    "query_cov": 1.0,
                    "model_cov": 1.0,
                    "identity": 1.0,
                    "id": "WP_000067268.1",
                    "db_xrefs": [
                        "NCBIProtein:WP_000067268.1"
                    ]
                }
            ],
            "genes": [
                "ant(9)-Ia"
            ],
            "id": "KIGIOPMKKF_3",
            "locus": "KIGIOP_00015"
        },
        {
            "type": "cds",
            "contig": "contig_1",
            "start": 1909,
            "stop": 2640,
            "strand": "-",
            "gene": "erm(A)",
            "product": "23S rRNA (adenine(2058)-N(6))-methyltransferase Erm(A)",
            "start_type": "ATG",
            "rbs_motif": "AGGAG",
            "db_xrefs": [
                "NCBIProtein:WP_001072201.1",
                "SO:0001217",
                "UniRef:UniRef50_P13956"
            ],
            "frame": 3,
            "aa": "MNQKNPKDTQNFITSKKHVKEILNHTNISKQDNVIEIGSGKGHFTKELVKMSRSVTAIEIDGGLCQVTKEAVNPSENIKVIQTDILKFSFPKHINYKIYGNIPYNISTDIVKRITFESQAKYSYLIVEKGFAKRLQNLQRALGLLLMVEMDIKMLKKVPPLYFHPKPSVDSVLIVLERHQPLISKKDYKKYRSFVYKWVNREYRVLFTKNQFRQALKHANVTNINKLSKEQFLSIFNSYKLFH",
            "aa_hexdigest": "20f866e6fc43b00080af44b66fd77b12",
            "nt": "ATGAACCAGAAAAACCCTAAAGACACGCAAAATTTTATTACTTCTAAAAAGCATGTAAAAGAAATATTGAATCACACGAATATCAGTAAACAAGACAACGTAATAGAAATCGGATCAGGAAAAGGACATTTTACCAAAGAGCTAGTCAAAATGAGTCGATCAGTTACTGCTATAGAAATTGATGGAGGCTTATGTCAAGTGACTAAAGAAGCGGTAAACCCCTCTGAGAATATAAAAGTGATTCAAACGGATATTCTAAAATTTTCCTTCCCAAAACATATAAACTATAAGATATATGGTAATATTCCTTATAACATCAGTACGGATATTGTCAAAAGAATTACCTTTGAAAGTCAGGCTAAATATAGCTATCTTATCGTTGAGAAGGGATTTGCGAAAAGATTGCAAAATCTGCAACGAGCTTTGGGTTTACTATTAATGGTGGAGATGGATATAAAAATGCTCAAAAAAGTACCACCACTATATTTTCATCCTAAGCCAAGTGTAGACTCTGTATTGATTGTTCTTGAACGACATCAACCATTGATTTCAAAGAAGGACTACAAAAAGTATCGATCTTTTGTTTATAAGTGGGTAAACCGTGAATATCGTGTTCTTTTCACTAAAAACCAATTCCGACAGGCTTTGAAGCATGCAAATGTCACTAATATTAATAAACTATCGAAGGAACAATTTCTTTCTATTTTCAATAGTTACAAATTGTTTCACTAA",
            "pscc": {
                "uniref50_id": "UniRef50_P13956",
                "db_xrefs": [
                    "SO:0001217",
                    "UniRef:UniRef50_P13956"
                ],
                "product": "rRNA adenine N-6-methyltransferase"
            },
            "expert": [
                {
                    "type": "amrfinder",
                    "rank": 95,
                    "gene": "erm(A)",
                    "product": "23S rRNA (adenine(2058)-N(6))-methyltransferase Erm(A)",
                    "method": "EXACTP",
                    "query_cov": 1.0,
                    "model_cov": 1.0,
                    "identity": 1.0,
                    "id": "WP_001072201.1",
                    "db_xrefs": [
                        "NCBIProtein:WP_001072201.1"
                    ]
                }
            ],
            "genes": [
                "erm(A)"
            ],
            "id": "KIGIOPMKKF_4",
            "locus": "KIGIOP_00020"
        }
    ],
    "sequences": [
        {
            "id": "contig_1",
            "description": "[gcode=11] [topology=linear]",
            "sequence": "TAACAACGGGATGCCCCAGCATATTGTGCAGAAATTTTTGGGGCATGAAAGCCCAGAAATGACAAGCAGATACGCTCATATCTTTGATGAAACTCTAAAAAATGAATTTACTAAATTTCAGGAAAAACTGGTTACCAATAATGGAGATGTGCTTGATCTAGATGAAGATAATGAAGTCGATGATGTAGAGCTTCAATGGTTCAAGAAAAATATAAATGCACAAGTGCTTCCAAATGGTTATTGTAGATTGCCAGTAGTAGCAGGTGGTTGTCCACATGCGAATGCATGCTTAGATTGCACTCACTTCTGTACCAGTAAGCAATTCTTACCACAGCACGAAGAACAGTTAGAGCGTACAGAAGAGTTATTAGCCATAGCTAAGGATAAACAATGGCAAAGACAAGTAGAGACTAATAGCCGTGTTAAAGAGCGTTTAGAACAAATCATTGGAAGTTTGACGGGGTAATTATCAATGGATAAACAAGTTAGAAATACAACAGAAATTGTACGTTTGGCGAAGCAGAAATCAAAAAAGACAAGGGAAAAAGTAGACAAAGCGATTTCTAAATTTTCGATTGAAGGTAAAGTTATTAATTTTAATTCAATAGCAAAGGAAGCTAATGTTTCTAAATCATGGCTTTATAAGGAACACGATATTAGGCAAAGAATCGAATCCCTTCGTGAGCGTCAAATAACAGCAAATGTAGTCTCAAAACCCAAGAAAAGTTCTCGTTCGGAGGAAATCCTTATTAAAACCTTAAAAAGAAGAGTAATGGAATTAGAAAAAGAAAATAAAAAATTACAGAACCAAATTCAAAAATTATATGGAGATCTGTATAATAAAGAATAATTATTAATCTGTAGACAAATTGTGAAAGGATGTACTTAAACGCTAACGGTCAGCTTTATTGAACAGTAATTTAAGTATATGTCCAATCTAGGGTAAGTAAATTGAGTATCAATATAAACTTTATATGAACATAATCAACGAGGTGAAATCATGAGCAATTTGATTAACGGAAAAATACCAAATCAAGCGATTCAAACATTAAAAATCGTAAAAGATTTATTTGGAAGTTCAATAGTTGGAGTATATCTATTTGGTTCAGCAGTAAATGGTGGTTTACGCATTAACAGCGATGTAGATGTTCTAGTCGTCGTGAATCATAGTTTACCTCAATTAACTCGAAAAAAACTAACAGAAAGACTAATGACTATATCAGGAAAGATTGGAAATACGGATTCTGTTAGACCACTTGAAGTTACGGTTATAAATAGGAGTGAAGTTGTCCCTTGGCAATATCCTCCAAAAAGAGAATTTATATACGGTGAGTGGCTCAGGGGTGAATTTGAGAATGGACAAATTCAGGAACCAAGCTATGATCCTGATTTGGCTATTGTTTTAGCACAAGCAAGAAAGAATAGTATTTCTCTATTTGGTCCTGATTCTTCAAGTATACTTGTCTCCGTACCTTTGACAGATATTCGAAGAGCAATTAAGGATTCTTTGCCAGAACTAATTGAGGGGATAAAAGGTGATGAGCGTAATGTAATTTTAACCCTAGCTCGAATGTGGCAAACAGTGACTACTGGTGAAATTACCTCGAAAGATGTCGCTGCAGAATGGGCTATACCTCTTTTACCTAAAGAGCATGTAACTTTACTGGATATAGCTAGAAAAGGCTATCGGGGAGAGTGTGATGATAAGTGGGAAGGACTATATTCAAAGGTGAAAGCACTCGTTAAGTATATGAAAAATTCTATAGAAACTTCTCTCAATTAGGCTAATTTTATTGCAATAACAGGTGCTTACTTTTAAAACTACTGATTTATTGATAAATATTGAACAATTTTTGGGAAGAATAAAGCGTCCTCTTGTGAAATTAGAGAACGCTTTATTACTTTAATTTAGTGAAACAATTTGTAACTATTGAAAATAGAAAGAAATTGTTCCTTCGATAGTTTATTAATATTAGTGACATTTGCATGCTTCAAAGCCTGTCGGAATTGGTTTTTAGTGAAAAGAACACGATATTCACGGTTTACCCACTTATAAACAAAAGATCGATACTTTTTGTAGTCCTTCTTTGAAATCAATGGTTGATGTCGTTCAAGAACAATCAATACAGAGTCTACACTTGGCTTAGGATGAAAATATAGTGGTGGTACTTTTTTGAGCATTTTTATATCCATCTCCACCATTAATAGTAAACCCAAAGCTCGTTGCAGATTTTGCAATCTTTTCGCAAATCCCTTCTCAACGATAAGATAGCTATATTTAGCCTGACTTTCAAAGGTAATTCTTTTGACAATATCCGTACTGATGTTATAAGGAATATTACCATATATCTTATAGTTTATATGTTTTGGGAAGGAAAATTTTAGAATATCCGTTTGAATCACTTTTATATTCTCAGAGGGGTTTACCGCTTCTTTAGTCACTTGACATAAGCCTCCATCAATTTCTATAGCAGTAACTGATCGACTCATTTTGACTAGCTCTTTGGTAAAATGTCCTTTTCCTGATCCGATTTCTATTACGTTGTCTTGTTTACTGATATTCGTGTGATTCAATATTTCTTTTACATGCTTTTTAGAAGTAATAAAATTTTGCGTGTCTTTAGGGTTTTTCTGGTTCATTATAACCTTCTCCTTACTGGTTCATTATAACCTTCTCCTTACTGGTTATAATGAACTGACATTAAGCGTTCATTATAACCAATTAATTTTGATTTGGTTGATAACGAACTTTATTAATAACAAATATAGAAAAAGTACCCATA",
            "length": 2783,
            "complete": false,
            "type": "contig",
            "topology": "linear",
            "simple_id": "contig_1",
            "orig_id": "sequence:",
            "orig_description": "sequence:"
        }
    ],
    "run": {
        "start": "2023-08-31 19:05:46",
        "end": "2023-08-31 19:06:00"
    },
    "version": {
        "bakta": "1.8.1",
        "db": {
            "version": "5.0",
            "type": "light"
        }
    }
}